ros/src/computing/perception/detection/packages/lidar_tracker/nodes/obj_fusion/obj_fusion.cpp

```diff
@@ -56,54 +58,80 @@ static double euclid_distance(const geometry_msgs::Point pos1,
 } /* static double distance() */
 
 /* fusion reprojected position and pointcloud centroids */
-static void fusion_objects(void) {
-  obj_label_t obj_label_current;
-  std::vector<autoware_msgs::CloudCluster> v_cloud_cluster_current;
+void fusion_cb(const autoware_msgs::obj_label::ConstPtr &obj_label_msg,
+  const autoware_msgs::CloudClusterArray::ConstPtr &in_cloud_cluster_array_ptr) {
```

Comment:
Yes, there braces go into a new line.
We are suggesting developers to use clang-format with ros style.
https://github.com/davetcoleman/roscpp_code_format